app/controllers/wiki_pages_controller.rb

```diff
@@ -102,6 +102,11 @@ def show
         js_data[:ModuleSequenceFooter_data] = item_sequence_base(Api.api_type_to_canvas_name('ModuleItem'), params[:module_item_id])
       end
 
+      context_module = @context.context_modules.first
```

Comment:
Can you add a comment here about what this code is for.  Maybe a small example of the contents of ```module_listing_data``` with context about the intention of auto creating a TOC of the modules.  If someone stumbles upon this code in the future, they will have a really hard time tracking down what it is for without a comment directing them to its usage.